frontend/src/components/DevTools.jsx

```diff
@@ -0,0 +1,271 @@
+import React, { useEffect, useState } from 'react';
+import Navbar from './Navbar';
+import '../styles/DevTools.css';
+
+const fillerWords = ["is", "are", "now", "the", "and", "a", "an", "in", "on", "to", "from", "for", "with", "of", "at", "by", "but", "or", "nor", "so", "too", "very", "can", "could", "will", "would", "should", "shall", "may", "might", "must", "ought"];
+
+const DevTools = () => {
+    const [overwatchData, setOverwatchData] = useState([]);
+    const [leagueData, setLeagueData] = useState([]);
+    const [error, setError] = useState('');
+    const [words, setWords] = useState([]);
+    const [keywords, setKeywords] = useState([]);
+    const [classifiers, setClassifiers] = useState([]);
+    const [nerfs, setNerfs] = useState([]);
+    const [buffs, setBuffs] = useState([]);
+    const [selectingFor, setSelectingFor] = useState(null);
+    const [selectedKeyword, setSelectedKeyword] = useState(null);
+
+    useEffect(() => {
+        const fetchAllData = async () => {
+            try {
+                const [overwatchResponse, leagueResponse] = await Promise.all([
+                    fetch('http://localhost:3000/patchnotes/overwatch/'),
+                    fetch('http://localhost:3000/patchnotes/league-of-legends/')
+                ]);
+
+                if (!overwatchResponse.ok || !leagueResponse.ok) {
+                    throw new Error('Network response was not ok');
+                }
+
+                const overwatchData = await overwatchResponse.json();
+                const leagueData = await leagueResponse.json();
+
+                console.log('Overwatch data:', overwatchData);
+                console.log('League data:', leagueData);
+
+                setOverwatchData(overwatchData);
+                setLeagueData(leagueData);
+
+                const extractedWords = extractWords(overwatchData, leagueData);
+                setWords(extractedWords);
+                console.log('Extracted words length:', extractedWords.length);
+
+            } catch (error) {
+                console.error('Error fetching patch details:', error);
+                setError('Failed to fetch patch details. Please try again later.');
+            }
```

Comment:
same thing here, it's not super convenient to split it out but whatever you can fetch/process should still happen regardless of other pieces failing, and debugging will be easier if you know exactly what is failing.